src/App.tsx

```diff
@@ -31,27 +31,31 @@ const SyncProviderPage = lazy(() => import('./pages/SyncProviderPage'))
 
 interface URLParams {
   provider: string
+  to: string
```

Comment:
I think this line is unnecessary

Reply:
thanks for the catch! forgot to remove that